src/contrast/deltaPhi.js

```diff
@@ -0,0 +1,34 @@
+// Delta Phi Star perceptual lightness contrast
+// The (difference between two Lstars each raised to phi) raised to (1/phi)
+// Symmetric, does not matter which is foreground and which is background
+
+
+import getColor from "../getColor.js";
+import {getLuminance} from "../luminance.js";
+
+// from CIE standard, which now defines these as rational fractions
```

Comment:
There is no need to re-implement CIE L*, it is already available in color.js.As an example usage, see

https://github.com/LeaVerou/color.js/blob/main/src/contrast/Lstar.js

Reply:
But it will need to be imported:

import lab from "../spaces/lab.js";

Reply:
Hi @svgeesus -- I didn't use the lab.js as it appeared to me that it does a transform to  D50, and DeltaPhiStar requires D65 for sRGB. I thought of adding a commit to adjust lab.js to have a D65 option, but I didn't want to open that can of worms...

Reply:
I added lab-d65.js in commit [fe321fa], and updated deltaPhi.js to match

Reply:
Aha yes, good point. Thanks for adding the D65 variant.